### PR TITLE
feat(beagle-analysis): add strategy-interview and strategy-review skills

### PR DESCRIPTION
**Pull request by anderskev**
## Summary

Two new skills for strategic planning in Claude Code, plus humanize skill improvements.

### Strategy Interview

Guided conversation that produces a strategy document using Richard Rumelt's kernel framework (diagnosis → guiding policy → coherent actions). Claude interviews you, pushes back on fluff and goals-masquerading-as-strategy, and only writes the document after the thinking holds up.

Three lenses load when the conversation warrants them:

- **Landscape mapping** (Wardley) — maps the value chain and component evolution before you make bets
- **Choice cascade** (Playing to Win) — forces specificity on where and how to compete
- **Value innovation** (Blue Ocean) — asks whether you should compete on these terms at all

Four-phase interview with gates that prevent premature document generation. Sessions persist to `.beagle/strategy/` for cross-session resume.

### Strategy Review

Evaluates an existing strategy document against seven dimensions. Conditionally applies four review lenses (McKinsey 7S, Balanced Scorecard, Porter's Five Forces, Hoshin Kanri). Findings are classified by severity with concrete recommendations.

**Judge artifact mode**: Produces machine-readable JSON (`strategy-review.json`) alongside prose for evaluation pipelines, RL loops, and LLM-as-judge workflows. Schema includes dimension scores, evidence provenance, bad-strategy patterns, failure paths, assumption risk map, and a deterministic pass/fail reward signal. 16 validation rules enforce consistency.

**Interview lens audit**: When a strategy was built with strategy-interview lenses (Wardley, Playing to Win, Blue Ocean), the review audits whether those lens findings survived into the draft or were dropped during document polishing.

Use strategy-interview to build a strategy. Use strategy-review to stress-test one.

### Also in this PR

- **Humanize skill**: Six new safe-fix categories (em dash overuse, title-case headings, curly quotes, three formulaic prose patterns). Reference material split into dedicated files for faster loading.
- **Pressure tests**: Validation scenarios for both skills covering edge cases, lens interactions, and judge artifact mode.
- **Cross-agent review**: Both strategy skills were reviewed by a second AI agent. Accuracy corrections, consistency fixes, and schema gaps closed from that feedback.
- Marketplace metadata: "strategy" and "strategic-planning" keywords. Skill count 124 → 126.

## Test plan

- [ ] `/beagle-analysis:strategy-interview` launches four-phase interview, persists state, resumes from prior session
- [ ] `/beagle-analysis:strategy-review` evaluates a strategy document against seven dimensions, applies lenses conditionally
- [ ] `/beagle-analysis:strategy-review` with "produce JSON" activates judge artifact mode, emits valid `strategy-review.json`
- [ ] Judge artifact JSON passes all 16 validation rules (score-label consistency, evidence provenance, weighted total arithmetic)
- [ ] Interview lens audit flags dropped findings when notes contain Wardley/cascade/value-innovation analysis
- [ ] `/beagle-docs:humanize` loads reference files and applies new safe-fix categories
- [ ] Skills appear in Claude Code skill list after plugin reload

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: plugins/beagle-analysis/skills/strategy-interview/SKILL.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
+---
+name: strategy-interview
+description: "Use when developing, critiquing, or pressure-testing a strategy for a company, product, team, career, or initiative. Triggers on: strategic planning, strategy doc, where-to-play/how-to-win choices, competitive positioning, market mapping, value innovation, blue ocean, uncontested market space, diagnosing goals masquerading as strategy."
+user-invocable: true
+---
```
[the remark this answers is not in the document]
Comment (anderskev) [reply plugins/beagle-analysis/skills/strategy-interview/SKILL.md:5]: @coderabbitai These are not workflow skills — they're conversational domain skills designed to auto-trigger when users discuss strategy topics. The `brainstorm` skill (same plugin, same pattern) also omits `disable-model-invocation`. Skills that have it (`llm-judge`, `write-adr`, `12-factor-apps-analysis`) are procedural command workflows that only make sense when explicitly invoked. Strategy skills use description-based trigger keywords and should remain model-invocable.

File: plugins/beagle-analysis/skills/strategy-review/SKILL.md [hunk 2]
```diff
@@ -0,0 +1,267 @@
+---
+name: strategy-review
+description: "Review and pressure-test strategy documents for kernel integrity, bad-strategy patterns, coherence gaps, and untested assumptions. Use when the user asks to review, critique, evaluate, or stress-test a strategy document, strategy-draft.md, strategy-notes.md, strategic plan, or any document claiming to be a strategy. Also triggers on: 'is this strategy any good', 'what's weak in this strategy', 'poke holes in this', 'red team this strategy', 'second opinion on our strategy'. Complements the strategy-interview skill — use strategy-interview to build a strategy through conversation, use strategy-review to evaluate one that already exists."
+user-invocable: true
+---
```
[the remark this answers is not in the document]
Comment (anderskev) [reply plugins/beagle-analysis/skills/strategy-review/SKILL.md:5]: @coderabbitai Same as the strategy-interview thread — these are conversational domain skills, not procedural workflows. They use description-based trigger keywords and match the `brainstorm` skill pattern (no `disable-model-invocation`). Skills that have this flag (`llm-judge`, `write-adr`) are command-style workflows only meaningful when explicitly invoked.

File: plugins/beagle-analysis/skills/strategy-review/SKILL.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+
+# Strategy Review
+
+Pressure-test strategy documents to find where they'll break before reality does it for you. The primary job isn't to evaluate prose quality or check formatting — it's to find the gaps, hidden failure paths, and under-accounted risks that will kill the strategy in execution. A strategy that survives this review has a meaningfully better chance of surviving contact with the real world.
+
+This skill complements the `strategy-interview` skill. Strategy-interview helps *build* a strategy through guided conversation; strategy-review subjects an existing strategy to rigorous adversarial evaluation using the same kernel framework (diagnosis, guiding policy, coherent actions) and bad-strategy filter.
```
[the remark this answers is not in the document]
Comment (anderskev) [reply plugins/beagle-analysis/skills/strategy-review/SKILL.md]: @coderabbitai Fixed in 254084e.

File: README.md [hunk 4]
```diff
@@ -110,6 +110,8 @@ These are the canonical skill entry points for Beagle.
 | `12-factor-apps-analysis` | beagle-analysis | 12-Factor compliance check |
 | `llm-judge` | beagle-analysis | Compare implementations |
 | `write-adr` | beagle-analysis | Generate ADRs from decisions |
+| `strategy-interview` | beagle-analysis | Build strategy through guided conversation |
+| `strategy-review` | beagle-analysis | Pressure-test existing strategy documents |
```
[the remark this answers is not in the document]
Comment (anderskev) [reply README.md:114]: @coderabbitai The entire README table (20+ entries) uses bare skill names with the plugin as a separate column. Qualifying only the new entries would be inconsistent. A full table convention change is out of scope for this PR.

File: plugins/beagle-analysis/skills/strategy-review/references/interview-lens-audit.md [hunk 5]
```diff
@@ -0,0 +1,115 @@
+# Interview Lens Audit
+
+Audit criteria for checking whether interview lens findings survived into the strategy draft. Use this during Step 3b when the strategy document or companion notes show evidence that landscape mapping, strategic choice cascade, or value innovation lenses were applied during the strategy-interview.
```
[the remark this answers is not in the document]
Comment (anderskev) [reply plugins/beagle-analysis/skills/strategy-review/references/interview-lens-audit.md]: @coderabbitai Fixed in 9a41ddf.

File: plugins/beagle-analysis/skills/strategy-review/references/judge-artifact-schema.md [hunk 6]
```diff
@@ -0,0 +1,472 @@
...
+      "evidence": [
+        {
+          "text": "Won't build enterprise compliance features this year. Won't match BuildOps on breadth.",
+          "provenance": "doc_quote",
+          "location": "Guiding Policy"
+        }
+      ],
+      "recommendation": null
+    }
+  ],
```
[the remark this answers is not in the document]
Comment (anderskev) [reply plugins/beagle-analysis/skills/strategy-review/references/judge-artifact-schema.md:143]: @coderabbitai Added a note in 9a41ddf marking the example as a partial illustrative fragment with explicit cardinality requirements. Expanding to a fully valid example would add ~150 lines of repetitive structure for marginal benefit — the field reference table already documents the constraints.

File: plugins/beagle-analysis/skills/strategy-interview/SKILL.md [hunk 7]
```diff
@@ -0,0 +1,315 @@
...
+name: strategy-interview
+description: "Use when developing or refining a strategy through guided conversation for a company, product, team, career, or initiative. Triggers on: strategic planning, building a strategy, where-to-play/how-to-win choices, competitive positioning, market mapping, value innovation, blue ocean, uncontested market space, diagnosing goals masquerading as strategy. For reviewing or critiquing an existing strategy document, use strategy-review instead."
```
[the remark this answers is not in the document]
Comment (anderskev) [reply plugins/beagle-analysis/skills/strategy-interview/SKILL.md]: @coderabbitai Fixed in 254084e.

File: plugins/beagle-analysis/skills/strategy-interview/SKILL.md [hunk 8]
```diff
@@ -0,0 +1,220 @@
...
+
+# Strategy Interview
+
+Turn Claude into a strategy interviewer who helps the user produce a strategy document grounded in the kernel framework (diagnosis, guiding policy, coherent action), enhanced with three complementary lenses applied when the conversation warrants them. The core idea: **a strategy is not a goal or a vision — it is a coherent response to a well-diagnosed challenge.**
+
+The user runs this expecting a conversation, not a form. Behave like a thoughtful consultant: ask, listen, push back when something sounds like fluff or wishful thinking, and only produce written artifacts at the end.
+
+<hard_gate>
+Do NOT produce strategy-draft.md or strategy-notes.md until Phase 3 is genuinely complete and the user has confirmed the kernel elements. Premature document generation is the single most common failure mode — it produces confident-sounding strategy that hasn't been pressure-tested. Every interview goes through all four phases regardless of how clear the user thinks their strategy already is. "Clear" strategies are where unexamined assumptions do the most damage.
+</hard_gate>
```
[the remark this answers is not in the document]
Comment (anderskev) [reply plugins/beagle-analysis/skills/strategy-interview/SKILL.md]: @coderabbitai Fixed in 2e984c1.

File: plugins/beagle-analysis/skills/strategy-review/SKILL.md [hunk 9]
```diff
@@ -0,0 +1,196 @@
...
+
+### Calibrate to the document's maturity
+
+- **Early draft / working document**: Focus on the big structural issues. Don't nitpick prose or completeness — the author knows it's rough. Emphasize what's promising and what needs the most work.
+- **Near-final / about to be shared**: Higher bar. Flag everything that could undermine credibility with the audience. Check that the "At a glance" section accurately represents the full document. Verify that claimed exclusions in the guiding policy aren't quietly reintroduced in the actions.
+- **Post-hoc / strategy already in execution**: Focus on assumption validation. Which assumptions can now be checked against reality? Flag any that have already been falsified by events.
+
+### Don't rewrite the strategy
+
+The review identifies problems and recommends fixes. It does not produce an alternative strategy. If the diagnosis is fundamentally wrong, say so and explain why — but the user needs to rethink it themselves (ideally using the strategy-interview skill). A reviewer who rewrites the strategy is doing the author's thinking for them, which means the author won't own it.
```
[the remark this answers is not in the document]
Comment (anderskev) [reply plugins/beagle-analysis/skills/strategy-review/SKILL.md]: @coderabbitai Fixed in 254084e.

File: plugins/beagle-analysis/skills/strategy-review/references/judge-artifact-schema.md [hunk 10]
```diff
@@ -0,0 +1,474 @@
...
+      "specificity_falsifiability": 10
+    },
+    "label": "Adequate",
+    "confidence": "high"
+  },
+
+  "reward_signal": {
+    "pass": false,
+    "score": 0.61,
+    "rationale": "One critical blocking finding (unverified capability assumption on load-bearing action). Strategy is structurally sound but not ready to act on without resolving the capability question."
```
[the remark this answers is not in the document]
Comment (anderskev) [reply plugins/beagle-analysis/skills/strategy-review/references/judge-artifact-schema.md:286]: @coderabbitai Fixed in `50aa4d8`. Score corrected to 0.82 per formula `(weighted_total - 1.0) / 3.0`.

File: plugins/beagle-analysis/skills/strategy-review/references/judge-artifact-schema.md [hunk 11]
```diff
@@ -0,0 +1,474 @@
...
+| `schema_version` | string | yes | Schema version. Currently `"1.0"`. |
+| `evaluator` | object | yes | Skill identifier and version. |
+| `review_metadata` | object | yes | Document metadata and review context. |
+| `kernel_extraction` | object | yes | Extracted kernel elements with evidence. |
+| `dimension_ratings` | array[7] | yes | All seven dimensions, each with score and evidence. |
+| `bad_strategy_patterns` | array[5] | yes | All five patterns, each with detected flag. Always include all five — `detected: false` for absent patterns. |
+| `assumption_risk_map` | array | yes | Load-bearing assumptions. May be empty if none found (unusual). |
+| `failure_paths` | array | yes | Failure scenarios. Minimum 2, typically 3–5. |
+| `review_lenses_applied` | array | no | Omit if no review lenses triggered. |
+| `interview_lens_audit` | array | no | Omit if no interview lenses were used during the strategy-interview. |
```
[the remark this answers is not in the document]
Comment (anderskev) [reply plugins/beagle-analysis/skills/strategy-review/references/judge-artifact-schema.md]: @coderabbitai Fixed in `afb59ce`–`4fab7a4`. Reference now uses `beagle-analysis:strategy-interview`.

File: docs/testing/test-plan.yaml [hunk 12]
```diff
@@ -0,0 +1,430 @@
...
+
+setup:
+  stack:
+    - type: markdown
+      package_manager: none
+  prerequisites:
+    - name: Python 3 (for YAML/JSON validation)
+      check: "python3 --version"
+    - name: Git
+      check: "git --version"
```
[the remark this answers is not in the document]
Comment (anderskev) [reply docs/testing/test-plan.yaml]: @coderabbitai File removed in `afb59ce`. The test plan was a generated artifact that shouldn't have been committed.

File: docs/testing/test-plan.yaml [hunk 13]
```diff
@@ -0,0 +1,430 @@
...
+      - run: |
+          wc -l \
+            plugins/beagle-analysis/skills/strategy-interview/SKILL.md \
+            plugins/beagle-analysis/skills/strategy-review/SKILL.md \
+            plugins/beagle-docs/skills/humanize/SKILL.md
+    expected: |
+      All three files are under 500 lines.
+      strategy-interview: ~317 lines
+      strategy-review: ~336 lines
+      humanize: ~281 lines
```
[the remark this answers is not in the document]
Comment (anderskev) [reply docs/testing/test-plan.yaml]: @coderabbitai File removed in `afb59ce`. The test plan was a generated artifact that shouldn't have been committed.

File: docs/testing/test-plan.yaml [hunk 14]
```diff
@@ -0,0 +1,430 @@
...
+          min_bytes = 500
+          for ref in refs:
+              size = os.path.getsize(ref)
+              name = '/'.join(ref.split('/')[-3:])
+              if size >= min_bytes:
+                  print(f'OK: {name} ({size:,} bytes)')
+              else:
+                  print(f'WARN: {name} only {size} bytes — may be too thin')
+          print(f'\nChecked {len(refs)} reference files (min {min_bytes} bytes)')
+          "
```
[the remark this answers is not in the document]
Comment (anderskev) [reply docs/testing/test-plan.yaml]: @coderabbitai File removed in `afb59ce`. The test plan was a generated artifact that shouldn't have been committed.

File: docs/testing/test-plan.yaml [hunk 15]
```diff
@@ -0,0 +1,430 @@
...
+              'plugins/beagle-analysis/skills/strategy-review',
+              'plugins/beagle-docs/skills/humanize',
+          ]:
+              name = skill.split('/')[-1]
+              orphans = check_orphans(skill)
+              if orphans:
+                  print(f'ORPHANS in {name}: {orphans}')
+              else:
+                  print(f'OK: {name} — no orphan references')
+          "
```
[the remark this answers is not in the document]
Comment (anderskev) [reply docs/testing/test-plan.yaml]: @coderabbitai File removed in `afb59ce`. The test plan was a generated artifact that shouldn't have been committed.
